### DISCUSSION TITLE
BibId Suggestion for any New Command to Replace \cite

### DISCUSSION THREAD

**Discussion by sdhosseini88** [James-Yu/LaTeX-Workshop#3755] [Q&A]
Hello everyone,

I'm using this great extension to write my report, and for a reason, I have defined a new command, for example, \m, which does the same work as \cite.
The actual reason for this is that I'm typing in a non-English language, and I want to avoid switching languages for every command.

But, after this redefinition, the suggestion of references will not appear for this new command.
Is there a way that we can tell this new command to show reference suggestions like \cite?

Thanks

**Comment by jlelong** [comment]
Unfortunately, it is impossible.
